conda/base/context.py

```diff
@@ -1899,6 +1901,11 @@ def description_map(self):
                 Disable check for `repodata.json.zst`; use `repodata.json` only.
                 """
             ),
+            envvars_force_uppercase=dals(
+                """
+            Force uppercase for new environment variable name. Defaults to True.
+            """
```

Comment:
```suggestion
                Force uppercase for new environment variable name. Defaults to True.
                """
```

Just some alignment :D